CONTRIBUTING.md

```diff
@@ -1,7 +1,71 @@
+# Rationale
+
+Contributions are always welcome. We accept [bug fixes](#bugs),
+[proposals and new ideas](#proposal), or simply [editorial
+improvements](#editorial-improvements).
+
+By submitting a pull request you are implicitly adhering to the
+[contribution agreement](#contribution-agreement)
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/acle/issues). You can also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/acle/pulls).
+
+## Proposals
+
+We are open to any proposal, including the introduction of new sets of
+intrinsics. We _do not_ require copyright assignment for the
+introduction of new specifications. The original contributor will
+retain the copyright.
+
+For adding new intrinsics or preprocessor macros, we recommend
+presenting them in a [pull
+request](https://github.com/ARM-software/acle/pulls), together with
+the following information:
+
+1. A rationale covering why these intrinsics or macros are needed.
```

Comment:
I really like this :)

Nitpicking: I wonder if instead of "needed", "helpful" would be better?
Maybe not everything in the ACLE is technically really "needed", but hopefully everything in it is "helpful"?